sklearn/ensemble/bagging.py

```diff
@@ -30,10 +30,14 @@
 MAX_INT = np.iinfo(np.int32).max
 
 
-def _generate_indices(random_state, bootstrap, n_population, n_samples):
+def _generate_indices(random_state, bootstrap, n_population, n_samples,
+                      feature_weight=None):
```

Comment:
In this context, "feature" is irrelevant and inappropriate. Maybe just call it p